### PR TITLE
Docs: Add Loki log context preselected labels section

### PR DESCRIPTION
**Pull request by ivanahuckova**
Documentation for following feature https://github.com/grafana/grafana/pull/68700

### CODE REVIEW

File: docs/sources/datasources/loki/query-editor/index.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ In Explore, you can can retrieve the context surrounding your log results by cli
 
 The initial log context query is created from all labels defining the stream for the selected log line. You can use the log context query editor to widen the search by removing one or more of the label filters from log stream. Additionally, if you used a parser in your original query, you can refine your search by using extracted labels filters.
 
+To reduce the repetition of selecting and removing the same labels when examining multiple log context windows, we store your updated labels and apply them to the next log context view you open. This allows you to seamlessly navigate through various log context windows without having to manually reapply the same filters repeatedly.
+
+At any time, if you would like to reset filters and use the initial log context query, you can use `Revert to initial query` button positioned next to the query preview and click it. This action will restore the original query, allowing you to start fresh.
```
Comment (matyax) [inline docs/sources/datasources/loki/query-editor/index.md:110]:
```suggestion
At any time, if you would like to reset filters and use the initial log context query, you can use the `Revert to initial query` button positioned next to the query preview. Clicking this button will restore the original query, allowing you to start fresh.
```

**Review by matyax — APPROVED** [review]
LGTM!

File: docs/sources/datasources/loki/query-editor/index.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ In Explore, you can can retrieve the context surrounding your log results by cli
 
 The initial log context query is created from all labels defining the stream for the selected log line. You can use the log context query editor to widen the search by removing one or more of the label filters from log stream. Additionally, if you used a parser in your original query, you can refine your search by using extracted labels filters.
 
+To reduce the repetition of selecting and removing the same labels when examining multiple log context windows, we store your updated labels and apply them to the next log context view you open. This allows you to seamlessly navigate through various log context windows without having to manually reapply the same filters repeatedly.
```
Comment (JStickler) [inline docs/sources/datasources/loki/query-editor/index.md:108]:
```suggestion
To reduce the repetition of selecting and removing the same labels when examining multiple log context windows, Grafana stores your selected labels and applies them to each open context window. This lets you seamlessly navigate through various log context windows without having to reapply your filters.
```

File: docs/sources/datasources/loki/query-editor/index.md [hunk 3]
```diff
@@ -105,6 +105,10 @@ In Explore, you can can retrieve the context surrounding your log results by cli
...
+
+At any time, if you would like to reset filters and use the initial log context query, you can use the `Revert to initial query` button positioned next to the query preview. Clicking this button will restore the original query, allowing you to start fresh.
```
Comment (JStickler) [inline docs/sources/datasources/loki/query-editor/index.md:110]:
```suggestion
To reset filters and use the initial log context query, click the `Revert to initial query` button next to the query preview.
```

**Review by JStickler — COMMENTED** [review]
[docs team] A couple of suggestions for more concise content.

**Review by JStickler — APPROVED** [review]
[docs team]  LGTM

**Pull request merged**